apps/meteor/client/views/omnichannel/businessHours/BusinessHoursForm.tsx

```diff
@@ -70,6 +70,9 @@ const BusinessHoursForm = ({ type }: { type?: 'default' | 'custom' }) => {
 						render={({ field }) => <Select id={timezoneField} {...field} options={timeZonesOptions} />}
 					/>
 				</FieldRow>
+				<Callout title={t('Daylight_savings_time')} type='info' mbs='x8'>
```

Comment:
Yup, indeed, we agreed with the Design team to show all of them since the validation for that to show only for DST active could be too expensive and unreliable, so we decided to rephrase the design suggestion with the agreement from them and show it for all of them.